src/lib/common/tag.cpp

```diff
@@ -208,48 +208,20 @@ std::string jsonInvalidCharsTransformation(const std::string& input)
 
 /* ****************************************************************************
 *
-* startTag1 -
+* startTag -
 */
-std::string startTag1
+std::string startTag
 (
   const std::string&  indent,
   const std::string&  key,
-  bool                showKey,
```

Comment:
About replacing 'bool showKey' with `key == ""` ...
I imagine showKey is FALSE if within vector and while it may be true that orion forces keys to be not empty, what about compound values?
Would be good to have a functest with compound values testing this.